files/logstash-configs/23-ufw.conf

```diff
@@ -1,23 +1,17 @@
 filter {
-  if [type] == "syslog" {
+  if [type] == "ufw" {
     grok {
       # Matches:
       # Feb 22 19:06:53 fpf-base-image kernel: [ 5431.910027] [UFW BLOCK] IN=eth0 OUT= MAC=04:01:ad:62:2d:01:3c:8a:b0:0d:6f:f0:08:00 SRC=50.233.105.123 DST=159.203.252.45 LEN=60 TOS=0x00 PREC=0x00 TTL=54 ID=57172 DF PROTO=TCP SPT=40169 DPT=3389 WINDOW=29200 RES=0x00 SYN URGP=0
-      match => { "message" => "%{SYSLOGBASE} \[ %{BASE10NUM}\] \[UFW %{WORD:action}] IN=%{WORD:ufw_interface} OUT= MAC=%{DATA:ufw_mac} SRC=%{DATA:ufw_src_ip} DST=%{DATA:ufw_dest_ip} LEN=%{BASE10NUM:length} TOS=%{DATA:tos} PREC=%{DATA:prec} TTL=%{BASE10NUM:ttl} ID=%{BASE10NUM:id}" }
-      add_tag => "ufw"
-    }
-  }
-  else if [type] == "ufw" {
-    grok {
       match => { "message" => "\[%{DATA}\] \[UFW %{WORD:ufw_action}\] IN=%{DATA:ufw_interface} OUT= MAC=%{DATA:ufw_mac} SRC=%{IP:ufw_src_ip} DST=%{IP:ufw_dest_ip} %{GREEDYDATA:ufw_tcp_opts} PROTO=%{WORD:ufw_protocol} SPT=%{INT:ufw_src_port} DPT=%{INT:ufw_dst_port} %{GREEDYDATA:ufw_tcp_opts}" }
```

Comment:
@conorsch part of me wants to update that var before we hook up the clients, but it's not a big deal, just a minor annoyance. your call.